app/components/audio/song-queue-card.tsx

```
@@ -0,0 +1,40 @@
+      />
+      <div className="flex-grow min-w-0">
+        <p className="font-medium truncate">{song.title}</p>
+        <p className="text-sm text-[--md-sys-color-outline] truncate">
+          {song.artists?.map(a => a.artist.name).join(', ')}
+        </p>
+      </div>
+      {onRemove && (
+        <button 
+          onClick={onRemove}
+          className="p-2 hover:bg-[--md-sys-color-surface] rounded-full"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)